docs/guide/agent-mode.md

```
@@ -221,6 +221,18 @@ List the contents of my Templates directory
 What files are in the attachments folder?
 ```
 
+#### get_workspace_state
+
+Get metadata about all files currently open in the editor. Returns each file's path, wikilink, whether it is visible in a pane, whether it is the active (focused) file, and any text the user has selected. Also includes the current project if the session is linked to one:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)